ansible/roles/pre_update_node/tasks/main.yml

```diff
@@ -1,16 +1,16 @@
 ---
-- name: Choose a single master to run commands on
-  set_fact:
-    cluster_master: "{{ groups['oo_hosttype_master'] | intersect(groups['oo_clusterid_' ~ oo_clusterid]) | first }}"
-  run_once: true
-  when: "{{ update_needed == True }}" 
-
-# FIXME: use the oadm_manage_node module instead
-- name: Make node unschedulable
-  command: "oadm manage-node {{ hostvars[oo_name].ec2_private_dns_name }} --schedulable=false"
-  register: unschedulable
-  until: unschedulable.stdout.find("SchedulingDisabled") != -1
-  retries: 10
-  delegate_to: "{{ cluster_master }}"
-  ignore_errors: True
-  when: "{{ update_needed == True }}" 
+#- name: Choose a single master to run commands on
+#  set_fact:
+#    cluster_master: "{{ groups['oo_hosttype_master'] | intersect(groups['oo_clusterid_' ~ oo_clusterid]) | first }}"
+#  run_once: true
+#  when: "{{ update_needed == True }}" 
+#
+## FIXME: use the oadm_manage_node module instead
+#- name: Make node unschedulable
+#  command: "oadm manage-node {{ hostvars[oo_name].ec2_private_dns_name }} --schedulable=false"
+#  register: unschedulable
+#  until: unschedulable.stdout.find("SchedulingDisabled") != -1
+#  retries: 10
+#  delegate_to: "{{ cluster_master }}"
+#  ignore_errors: True
+#  when: "{{ update_needed == True }}" 
```

Comment:
I like the example of setting the master that you showed me. I'll try to implement that.

You could be right that the role is not necessary. But it's possible that we'll be left with a one-task role until `when` statements work with `include_role`. I'll see if I can address both these issues.

There's one part of your comment that I didn't understand: 'We should use "node" instead of selector in this instance'. What do you mean by that? 'selector' doesn't appear in the above code that I can see.